### DISCUSSION TITLE
StorageD design docs

### DISCUSSION THREAD

**Discussion by porscheme** [vesoft-inc/nebula#5359] [Q&A] [answered]
@wey-gu 

Can you please point me to StorageD design documents
Thanks

**Comment by QingZ11** [comment, marked as answer]
@porscheme You can read the docs, like: [https://docs.nebula-graph.io/3.4.0/1.introduction/3.nebula-graph-architecture/4.storage-service/](https://docs.nebula-graph.io/3.4.0/1.introduction/3.nebula-graph-architecture/4.storage-service/)

**Comment by wey-gu** [comment]
For now there are blog posts about how storage layer was designed here: https://www.nebula-graph.io/tags/architecture
